service/storage_service.cc

```diff
@@ -604,6 +604,26 @@ void storage_service::mark_existing_views_as_built() {
     }).get();
 }
 
+std::list<gms::inet_address> storage_service::get_ignore_dead_nodes_for_replace() {
+    std::vector<sstring> ignore_nodes_strs;
+    std::list<gms::inet_address> ignore_nodes;
+    boost::split(ignore_nodes_strs, _db.local().get_config().ignore_dead_nodes_for_replace(), boost::is_any_of(","));
+    for (std::string n : ignore_nodes_strs) {
+        try {
+            std::replace(n.begin(), n.end(), '\"', ' ');
+            std::replace(n.begin(), n.end(), '\'', ' ');
```

Comment:
Nitpick (don't have to fix now) I wish we didn't parse every list argument in a different function, and had a common function to parse all lists. How can a user guess that in this function list is supposed to be comma-separated, spaces are allowed around commas, and quotes are replaced by spaces - but in some other function there're different rules?